### PR TITLE
Update path to Elastic Agent repo in docs

### LINKED ISSUE #1743: [Request] Adjust any kubernetes template links to new `elastic-agent` repo
## Description

We've recently added new repo `elastic-agent` and we moved 
https://github.com/elastic/beats/tree/main/deploy/kubernetes/elastic-agent-managed
https://github.com/elastic/beats/tree/main/deploy/kubernetes/elastic-agent-standalone

to the new repo:
https://github.com/elastic/elastic-agent/tree/main/deploy/kubernetes

There might be some docs that are linking to these templates in the beats repo, if so that should be redirected to the new elastic agent repo.
https://www.elastic.co/guide/en/fleet/master/running-on-kubernetes-managed-by-fleet.html
https://www.elastic.co/guide/en/fleet/current/running-on-kubernetes-standalone.html

## Collaboration
*(Choose the expected collaboration model and delete the others.)* 

- The docs team will lead producing the content
- The product team will provide the initial content and the docs team will edit / review
- The docs team will define with product team the structure and location, and the product team will provide the initial content
- Other (please describe)

*(Assign a contact person for this issue. We need to have a contact person in the product/development team to provide information about how the item to be documented works. This can be omitted when the product/development team is providing the initial content, as the contact person will be the one making the initial contribution.)*

**Contact Person:**

## Suggested Target Release

*(Let us know if this feature is needed for a specific release.)*

## Stakeholders

*(Please list any stakeholders for this issue.)*

cc @dedemorton as you have handled the transition to the new repo before

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #1743 

I've updated the links. Please verify that the URLs are correct and let me know if anything else needs to be changed. I'm not sure when these topics were last verified for technical accuracy. Now might be a good time to do that.

### CODE REVIEW

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc [hunk 1]
```diff
@@ -45,7 +45,7 @@ To download the manifest file, run:
 
 ["source", "sh", subs="attributes"]
 ------------------------------------------------
-curl -L -O https://raw.githubusercontent.com/elastic/beats/{branch}/deploy/kubernetes/elastic-agent-managed-kubernetes.yaml
+curl -L -O https://raw.githubusercontent.com/elastic/elastic-agent/{branch}/deploy/kubernetes/elastic-agent-managed/elastic-agent-managed-daemonset.yaml
```
Comment (narph) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc:48]: @dedemorton , the file name should be the same as previous `elastic-agent-managed-kubernetes.yaml`, only the path from `beats` to `elastic-agent` changes. @ChrsMark can you confirm?

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc [hunk 2]
```diff
@@ -136,7 +136,7 @@ To deploy {agent} on Kubernetes, run:
 
 ["source", "sh", subs="attributes"]
 ------------------------------------------------
-kubectl create -f elastic-agent-managed-kubernetes.yaml
+kubectl create -f elastic-agent-managed-daemonset.yaml
```
Comment (narph) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc:139]: same as above

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc [hunk 3]
```diff
@@ -151,7 +151,7 @@ elastic-agent-olpsd   1/1     Running   0          12m
 ------------------------------------------------
 
 NOTE: You might need to adjust https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[resource limits] of the elastic-agent container
-in the `elastic-agent-managed-kubernetes.yaml` manifest. Container resource usage depends on the number of datastreams
+in the `elastic-agent-managed-daemonset.yaml` manifest. Container resource usage depends on the number of datastreams
```
Comment (narph) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc:154]: same as above

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc [hunk 4]
```diff
@@ -40,7 +40,7 @@ To download the manifest file, run:
 
 ["source", "sh", subs="attributes"]
 ------------------------------------------------
-curl -L -O https://raw.githubusercontent.com/elastic/beats/{branch}/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml
+curl -L -O https://raw.githubusercontent.com/elastic/elastic-agent/{branch}/deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-daemonset.yaml
```
Comment (narph) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc:43]: same as above

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc [hunk 5]
```diff
@@ -109,7 +109,7 @@ To deploy to Kubernetes, run:
 
 ["source", "sh", subs="attributes"]
 ------------------------------------------------
-kubectl create -f elastic-agent-standalone-kubernetes.yaml
+kubectl create -f elastic-agent-standalone-daemonset.yaml
```
Comment (narph) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc:112]: same as above

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc [hunk 6]
```diff
@@ -125,7 +125,7 @@ elastic-agent-hs4pb             1/1     Running   0          81m
 ------------------------------------------------
 
 NOTE: You might need to adjust https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[resource limits] of the elastic-agent container
-in the `elastic-agent-standalone-kubernetes.yaml` manifest. Container resource usage depends on the number of datastreams
+in the `elastic-agent-standalone-daemonset.yaml` manifest. Container resource usage depends on the number of datastreams
```
Comment (narph) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc:128]: same as above

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc (hunk 1 above)
Comment (ChrsMark) [reply docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc:48]: yeap, you need the actual file path here.

**Review by ChrsMark — CHANGES_REQUESTED** [review]
We need to refer to the manifests with their actual names/paths as we find them in the upstream repo.

**Review by ChrsMark — APPROVED** [review]
lgtm

**Pull request merged**